experimental/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 All notable changes to experimental packages in this project will be documented in this file.
 
+## New Feature 
+logs feature added in SDK-NODE
+
```

Comment:
This should go down in the unreleased section under `### :rocket: (Enhancement)`, please provide it in this format:

`* feat(sdk-node): logs support added [#3969](https://github.com/open-telemetry/opentelemetry-js/pull/3969) @psk001`
so that it is in-line with the other entries.